app/views/user_mailer/tester.html.erb

```diff
@@ -0,0 +1,4 @@
+<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
+<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> test email").html_safe %></h2>
+<h4><%= _("This email confirms Foreman email configuration.") %></h4>
```

Comment:
"This is a test message to confirm that Foreman's email configuration is working." ?